Navigate to chat when session is selected

Rossi1337 opened the pull request:
When a session is activated we straight away naviage to the "chat page" so that you can continue chatting in that session.
For this I pass now the pageController into the SessionPage

Additionally I changed the order of the buttons: First the "Enter Session" and then the "Delete". This is more safe as you do not have to move with the mouse over the Delete button to reach the Enter button.

WilliamKarolDiCioccio commented:
Good job! Can you do the same for models so we group everything into a single pull request?

Rossi1337 commented:
Yes I will do this

Rossi1337 commented:
Implemented now the same logic for the models. When one is activated we jump now to the chat.
Also changed the order of the buttons to match the sessions page again.
